### ISSUE TITLE
Need some sort of feedback when commands like channel join fail

### ISSUE THREAD

**Issue by TeoTwawki** [j0ack/discirc#5]
Ok so I've been running this all week, been pretty satisfied with it. Today suddenly after restarting the server it lives on, it can't join any IRC channels. I can't diagnose it because there are no errors at all. It simply never ducking joins. I can tell its actually connected by sending info requests and messaging it directly - its on the network, just not in channel. It connects to discord just fine. Even transmits private messages from discord to IRC just fine, Just doesn't join channel anymore. And I control the channel I am testing on - its not blocked, As I said it *was* working fine all week and I have changed nothing.

**Comment by j0ack** [comment]
Hello, 

Could you please post your config file here ?
If discirc can receive private messages, it means that connection works just fine. How did you install discirc ?

**Comment by TeoTwawki** [comment]
Its not a prob with the config, as the config hasn't changed.
Installation was fine, this was working all week long. This wasn't the first time I stopped and restarted it either.

It gets on and sets it nick but doesn't succeed in joining channel, there is zero feedback about why. Since nothing on my end has changed whatsoever, the rizon's end where the irc channel is hosted has to have changed something. Rizon staff isn't going to help me diagnose this, they expect me to read the server response to figure out what went wrong. The problem is I can't see that - its sent to the bot, and the bot doesn't show me.

Even if my bot magically begins working again, it would still be nice for the program to say something when it fails to join the IRC channel.

There is no feedback on any problems that aren't a crash basically. Like NickServ would respond with an message if something went wrong setting nick for example (like someone else has that nick registered already), but the bot isn't coded to do anything with these messages. I think they should be printed to the command line.

**Comment by j0ack** [comment]
You're absolutely right. There should be logging message errors concerning IRC/Discord commands.
I will try to add this feature asap to help users to debug the app.

**Comment by TeoTwawki** [comment]
Thanks. ~~Once I have my end working again, I have a few pull requests to send your way that extend functionality a bit.~~ #7

I found out someone swiped my bots nick, so I've added code to make it msg nickserv to identify to a protected nick now, and also added joining password protected channels while I was at it.

I got my bots nick back but rizon seems to have tied it to a my main nicks IP while they were at it so now it only signs in from my home pc instead of my server..~~I'll have to get that sorted out later this week.~~ _Rizon sucks._
